prow/README.md

```diff
@@ -1,71 +1,159 @@
 # Prow
 
-Prow dashboard: http://prow.apps.ci.metal3.io
+Metal3 Prow dashboard: https://prow.apps.test.metal3.io
```

Comment:
why `test`? why not `ci`?

Reply:
this is a new domain, previously it was http://prow.apps.ci.metal3.io, because Russell had DNS records pointing to his cluster IPs. We created a new record (`prow.apps.test.metal3.io` -> our IP)  with under the metal3.io domain to point to the new cluster.